addons/dotnet/dotnet.addon

```diff
@@ -0,0 +1,8 @@
+# Name: dotnet
+# Description: ImageStreams and Templates for developing and deploying application using .NET Core
+
+oc create --namespace openshift -f dotnet_imagestreams.json
```

Comment:
Don't use create, but apply. With create an addon can not be applied twice, with apply, there's no error if you apply the addon twice.

Reply:
@jorgemoralespou thanks for the tip.